db/wide/wide_columns_helper.h

```diff
@@ -34,7 +36,31 @@ class WideColumnsHelper {
     return columns.front().value();
   }
 
-  static void SortColumns(WideColumns& columns);
+  static void SortColumns(WideColumns& columns) {
+    std::sort(columns.begin(), columns.end(),
+              [](const WideColumn& lhs, const WideColumn& rhs) {
+                return lhs.name().compare(rhs.name()) < 0;
+              });
+  }
+
+  template <typename Iterator>
+  static Iterator Find(Iterator begin, Iterator end, const Slice& column_name) {
+    assert(std::is_sorted(begin, end,
```

Comment:
QQ. Does the check inside `assert()` get ignored automatically in non-debug build?

If not, I'm wondering if we want to wrap this assertion in `#ifndef NDEBUG`

Reply:
> QQ. Does the check inside `assert()` get ignored automatically in non-debug build?

Yes, it is a no-op in non-debug builds